omod/src/main/webapp/resources/scripts/dojo/src/widget/openmrs/OpenmrsSearch.js

```diff
@@ -475,9 +475,7 @@ dojo.widget.defineWidget(
 
 
 	noCell: function() {
-		var td = document.createElement("td");
```

Comment:
Did you intentionally remove this?

Reply:
Yes. By setting the display value of the element to "none" we get a table similar to this.

![Screenshot from 2024-10-02 19-49-16](https://github.com/user-attachments/assets/9361f86c-e521-4f2a-81b7-98db2e16d050)

Reply:
You mean it has been like that and no body has ever seen it? Or was it caused by something else?

Reply:
I think so. That's how HTML works.

```
    <table border="1">
        <tbody>
        <tr>
            <td>Hello</td>
            <td>World</td>
            <td>Hello</td>
        </tr>
        <tr>
            <td>Hello</td>
            <td style="display: none"></td>
            <td>Hello</td>
        </tr>
        </tbody>
    </table>
```

For example this HTML code outputs this:

![Screenshot from 2024-10-02 20-23-08](https://github.com/user-attachments/assets/c8e45efd-0123-4541-baa7-dcccaafa2a16)